201-IoT-IotHub-EdgeEmulator-VM/azuredeploy.json

```diff
@@ -0,0 +1,275 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "defaultValue": "EdgeVM",
+            "metadata": {
+                "description": "The name of you Virtual Machine."
+            }
+        },
+        "IoTHubname": {
+            "type": "string",
+            "defaultValue": "[concat('IoThub', resourceGroup().name)]",
+            "metadata": {
+                    "description": "Name for your IotHub"
+            }
+         },   
+        "location": {  
+		"type": "string",  
+            "defaultValue": "[resourceGroup().location]",  
+            "metadata": {  
+                    "description": "Location for all resources."  
+            }  
+        },  
+        "VmSize": {
+            "type": "string",
+            "defaultValue": "Standard_B1ls",
+            "allowedValues": [
```

Comment:
remove allowedValues

Reply:
Done, and also changed the GettingStarted.